src/HvLightPipe.c

```diff
@@ -20,11 +20,13 @@
 #include <xmmintrin.h>
 #define hv_sfence() _mm_sfence()
 #elif __arm__ || HV_SIMD_NEON
-  #if __ARM_ACLE
+  #ifdef __ARM_ACLE
     #include <arm_acle.h>
     // https://msdn.microsoft.com/en-us/library/hh875058.aspx#BarrierRestrictions
     // http://doxygen.reactos.org/d8/d47/armintr_8h_a02be7ec76ca51842bc90d9b466b54752.html
     #define hv_sfence() __dmb(0xE) /* _ARM_BARRIER_ST */
+  #elif defined ARM_CORTEX
+    #define hv_sfence() __ASM volatile ("dmb 0xF":::"memory")
```

Comment:
I've not got a definitive way to test this but `0xE` appears to work the same as `0xF`.
I'm still getting device crashes for reasons I don't understand.
I have a function called on MIDI interrupt which creates a HvMessage (on the stack) and schedule it for `notein`. Sending in lots of MIDI leads to occasional crashes.
The interrupt may be called at any time during or after the audio block process.